modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc

```diff
@@ -2235,11 +2253,11 @@ behaviour of the grace period.
 ....
 
 === Link to get a demo key during active grace period
-As admin you will be directed to that web page if you click on the "get a demo key"
+The admin will be directed to that web page if he clicks in the "get a demo key"
```

Comment:
We let a "he" in here - I have plenty of "she" friends who are excellent admins! I will make a core PR to fix this (and any other occurrences that I find) - and it will get fixed here the next time we transfer.